Formula/isync.rb

```diff
@@ -37,6 +37,45 @@ def install
     system "make", "install"
   end
 
+  def caveats; <<-EOS.undent
+    If you setup your ~/.mbsync file with a Group called "Periodic", 'brew services start isync' will sync that group every five minutes.  Typically, those rules would check your inbox, trash, and sent mail.
```

Comment:
Seems fine to me.  An isync user could figure out the "Periodic" group was what they wanted based on the plist.  If they can't figure that out, then the comment probably won't be much help.

Profoundly ignorant question: do I make another pull request with that change?  Or can you merely alter it?  (if the later, go ahead)